framework/src/errors.ts

```diff
@@ -15,6 +15,7 @@
 
 export class FrameworkError extends Error {
 	public name: string;
+	public code = 'framework';
```

Comment:
I will suggest to use standard error pattern as `NodeJS` itself. Please see following list of error codes which are available in NodeJS and we can use similar pattern in our case. 

e.g. For this one particular it will become `ERR_FRAMEWORK`

https://nodejs.org/dist/latest/docs/api/errors.html#errors_node_js_error_codes